### PR TITLE
Merge my work from over the weekend

### PR DESCRIPTION
**Pull request by tkpanke**
Merge my work from over the weekend with stacktrace

### CODE REVIEW

**Comment by OneLeggedPigeon** [comment]
@tkpanke did you check with @RevatureRobert/styling for you changes to style.css?

File: src/app/stacktrace/components/new-stacktrace/new-stacktrace.component.html
Comment (pickettmap) [inline src/app/stacktrace/components/new-stacktrace/new-stacktrace.component.html:2]: We aren't using Angular material. Change this to bootstrap card

**Review by OneLeggedPigeon — APPROVED** [review]
Everything looks good to me, but let's wait till styling reviews style.css before merging.

File: src/app/stacktrace/stacktrace.module.ts
Comment (pickettmap) [inline src/app/stacktrace/stacktrace.module.ts:6]: Remove this

File: src/styles.css
Comment (pickettmap) [inline src/styles.css:3]: Don't change the styles.css. For now, keep them on a component basis within your [component].css. If it shows that we do need to incorporate this as a global style, we can alter it then

File: src/index.html
Comment (pickettmap) [inline src/index.html:11]: Remove this

**Review by pickettmap — CHANGES_REQUESTED** [review]
Switch from angular materials to bootstrap. Remove the changes to styles.css and apply those rules on a component basis for now.

**Pull request merged**
